CalibCalorimetry/EcalTrivialCondModules/src/EcalLaserCondTools.cc

```diff
@@ -338,7 +332,7 @@ void EcalLaserCondTools::fillDb(CorrReader& r) {
 }
 
 void EcalLaserCondTools::processIov(CorrReader& r, int t1, int t2[EcalLaserCondTools::nLmes], int t3) {
-  static int iIov = 0;
+  int iIov = 0;
```

Comment:
No, this is not correct. As such it is initialized to 0, incremented to 1 in the line below, and stop.

To be honest, I also don't understand the original version (perhaps, someone responsible of this code should have a look at it...); but his "fix" is certainly wrong.

Either  fix  it correctly, or let it as it was and defer the correct fix to some other future PR

Reply:
I confirm that this change would break the code.

I recommend to keep the changes to the one required to fix the bug this PR is meant to fix. 

If the static is wanted to be removed for coding policy (but then why wasn't it a problem before this PR?!?), please open an issue and we will fix it in a separate PR.

Philippe.

Reply:
> I confirm that this change would break the code.
> 
> I recommend to limit the changes to what is required to fix the bug this PR is meant to fix.
> 
> If the static is wanted to be removed for coding policy (but then why wasn't it a problem before this PR?!?), please open an issue and we will fix it in a separate PR.
> 
> Philippe.

I agree, please revert the change for `iIow`
I only [requested](https://github.com/cms-sw/cmssw/pull/36768#issuecomment-1019338800) to fix the other static variable `done`, which was a clear bug independently of its static-ness